File: 下學期/W1/ExtraScore.py
import sys

for s in sys.stdin:
    if s == "":
        break
    if s != "\n":
        scores = list(map(int, s.split()))
        sum = 0
        for n in scores:
            sum += n
        print(sum)
# 6 9 420
#
# 74 14
# 214 74 386 47

File: 上學期/w10/ExtraScore.c
#include <stdio.h>
#include <string.h>

char s[1000005];
int main()
{

  while(fgets(s, sizeof(s), stdin))
  {
    int sum = 0;
    if(s[0] == '\n') continue;

    int n = 0;
    int len = strlen(s);
    for(int i = 0; i < len+1; i++)
    {
      if(s[i] == ' ' || s[i] == '\n')
      {
        sum += n;
        n = 0;
      }
      else
      {
        n *= 10;
        n += s[i] -'0';
      }
    }
    printf("%d\n", sum);
  }
  return 0;
}
